frontend/src/routes/(authenticated)/project/[project_code]/+page.ts

```diff
@@ -277,3 +277,26 @@ export async function _refreshProjectMigrationStatusAndRepoInfo(projectCode: str
     throw result.error;
   }
 }
+
+
+export async function _leaveProject(projectId: string): $OpResult<LeaveProjectMutation> {
+//language=GraphQL
+  const result = await getClient()
+  .mutation(
+    graphql(`
+      mutation LeaveProject($projectId: UUID!) {
```

Comment:
yeah about that, I just tried it and I'm running into similar issues that we had when deleting a project. Except this time when it refreshes there's an auth failure and we don't handle that gracefully since the front end is supposed to never call something the user doesn't have permission for.